docs/source/user guide/controller adding.rst

```diff
@@ -13,23 +13,37 @@ built it and need to flash it, read :doc:`controller flashing` and complete that
 .. warning:: Prior to setting up a controller with Fermentrack, please read any notes specific to your controller's hardware on the :doc:hardware page.
 
 
-Setting up a WiFi-connected controller using the Guided Workflow
----------------------------------------------------------------------
+Configuring an ESP8622 controller for WiFi
```

Comment:
I would read “using the Guided Workflow” to the end of this header to match the phrasing for the serial workflow

Reply:
I meant to say re-add, not read.

Reply:
Fixed